sdk/textanalytics/azure-ai-textanalytics/samples/async_samples/sample_analyze_sentiment_async.py

```diff
@@ -59,7 +59,10 @@ async def analyze_sentiment_async(self):
             ))
             for sentence in doc.sentences:
                 print("Sentence '{}' has sentiment: {}".format(sentence.text, sentence.sentiment))
-                print("Sentence confidence scores: positive={}; neutral={}; negative={}".format(
+                print("...Sentence is offset from the beginning of the document by {} characters and is {} long".format(
```

Comment:
it could be my translation, but `Sentence is offset from...` sounds weird. Consider rewording